SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialRPCService.cpp

```diff
@@ -83,6 +83,10 @@ EPushRPCResult SpatialRPCService::PushRPCInternal(Worker_EntityId EntityId, ERPC
 			LastAckedRPCId = GetAckFromView(EntityId, Type);
 		}
 	}
+	else if (bCreatedEntity)
+	{
```

Comment:
Please add a comment here describing when this happens.

Reply:
Added comment.